torchaudio/datasets/cmuarctic.py

```diff
@@ -76,9 +76,15 @@ def load_cmuarctic_item(line: str,
 
 
 class CMUARCTIC(Dataset):
-    """
-    Create a Dataset for CMU_arctic. Each item is a tuple of the form:
-    waveform, sample_rate, utterance, utterance_id
+    """Create a Dataset for CMU_ARCTIC.
+
+    Args:
+        root (str): Path to the directory where the dataset is found or downloaded.
+        url (str, optional):
+            Type of the dataset to dowload. This is **NOT** the actual URL. (default: ``"aew"``)
```

Comment:
How does a reader know which values they can use and what they do?

Reply:
Yeah, so that's the problem. I see the list of valid values like the followings and I wonder, what is the best way to add all of them???????

```
        if url in [
            "aew",
            "ahw",
            "aup",
            "awb",
            "axb",
            "bdl",
            "clb",
            "eey",
            "fem",
            "gka",
            "jmk",
            "ksp",
            "ljm",
	    "lnh",
            "rms",
            "rxr",
            "slp",
            "slt"
	]:
```

Reply:
okay added them as is.